doc/source/contrib/language_ref/property_ref/requirement_types.rst

```diff
@@ -95,16 +95,19 @@ Or with revision ranges as follows:
 
     snap:
       mypackage:
-        - min: 0.0
-          max: 1.0
-        - min: 4.0
-          max: 5.0
+        - min: 1234
+          max: 2345
+          channel: 2.0/stable
+        - min: 3456
+          max: 4567
 
-In the above example *mypackage* must have a revision between 0.0 and 1.0 or
-4.0 and 5.0 inclusive.
+In the above example *mypackage* must have a revision between *1234* and *2345*
+or 3456 and 4567 inclusive. If *mypackage* revision is between *1234* and
```

Comment:
nit: should be *3456* and *4567*

Reply:
the channel is only checked if the revision is within the corresponding range, so technically you can say:

```
   - min: 1
     max: 2
     channel: X
   - min: 3
     max: 4
     channel: Y
```

and for snap rev 1 it will ensure it came from channel X and snap rev 3 it will ensure it came from channel Y